pallets/inflation-rewards/src/lib.rs

```diff
@@ -119,7 +125,9 @@ pub mod pallet {
             // Let the runtime handle the non-staking part
             T::OnUnbalanced::on_unbalanced(not_distributed_rewards.merge(total_reminder));
 
-            weight += Self::reward_orchestrator_author();
+            if let Some(orchestrator_author) = T::GetSelfChainBlockAuthor::get_block_author() {
```

Comment:
comment: "we dont reward the orchestrator in solochain mode"

Reply:
_fixed_ in f117a242086e1c10409e64f069c79ec10d6b391c.